README.md

```diff
@@ -2,6 +2,13 @@
 
 ![](https://jlucassr.github.io/ArBC-Pages/imagens/logo.jpg)
 
+## Porquê do Nome?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)